modules/01-mods.js

```diff
@@ -50,6 +53,39 @@ async function slashModBan(interaction) {
   await c.ban(interaction, target, reason, days);
 }
 
+async function slashModFilter(interaction) {
+  await interaction.deferReply({ ephemeral: true });
+  const word = interaction.options.getString("word").toLowerCase().trim();
+  const user = interaction.member;
+  const modLogs = interaction.guild.channels.cache.get(sf.channels.modlogs);
+  const filtered = pf.scan(word);
+  const apply = interaction.options.getBoolean("apply") ?? true;
+  if (!user.roles.cache.has(sf.roles.management) && !user.roles.cache.has(sf.roles.manager) && !sf.adminId.includes(user.id)) {
```

Comment:
We can use the functions from `utils/perms` to slim this down a bit.
```suggestion
  if (!p.isMgmt(interaction) && !p.isMgr(interaction) && !p.isAdmin(interaction)) {
```